.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7.18]
```

Comment:
Pouzivas inu verziu pythonu, a povolime aj ine verzie?

Reply:
@viliambalaz Rovnaku ako na produkcii:
```
root@vmi324598:~# python --version
Python 2.7.15+
```

Verziu budeme zvysovat postupne ako budeme upgradovat na novsie verzie

Reply:
pre verziu 2.7.15 a 2.7.16 vypise chybu ```##[error]Version 2.7.16 with arch x64 not found```

Reply:
@viliambalaz Staci to pustit na jednej verzii z nich, netreba zbytocne oba. Ktorakolvek z nich je OK.

Reply:
:ok: